lfucg-exactions/server/accounts/helper.py

```diff
@@ -39,3 +42,42 @@ def send_lost_username_email(user):
     msg = EmailMultiAlternatives(subject, text_content, from_email, [user.email])
     msg.attach_alternative(html_content, "text/html")
     msg.send()
+
+def send_email_to_supervisors(**kwargs):
+    if 'perm_name' in kwargs:
+        perm = Permission.objects.get(codename=kwargs['perm_name'])
+
+    q_objects = Q()
+    for group in kwargs['group']:
+        q_objects.add(Q(groups__name=group), Q.OR)
+    users = User.objects.filter(Q(profile__is_supervisor=True), q_objects)
```

Comment:
you could just do users = User.objects.filter(Q(profile__is_supervisor=True) | Q(groups__name__in=group)) where group is the array parameter

Reply:
Hmm. That's interesting. Thanks!

Reply:
From what I understand @Wei-KeatKong what I doing here is saying the User's `is_supervisor=True &` the user is a part of one of the groups that are being passed in. That's what the `, q_objects` is saying. Is it implied that the groups would function as an OR in the one filter as an array parameter? Trying to set it up for the future if more than one group's supervisor were supposed to get the emails. Does that make sense?